lib/client.dart

```diff
@@ -275,6 +279,17 @@ class RouteImpl extends Route {
   }
 
   @override
+  void addRedirect({String name, Pattern path}) {
```

Comment:
that's a fair point.  i'll noodle on the naming...maybe `toRouteName` just to be extra explicit?